components/post_view/message_attachments/action_button/action_button.jsx

```diff
@@ -3,25 +3,36 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
+import {changeOpacity} from 'mattermost-redux/utils/theme_utils';
 
 import Markdown from 'components/markdown';
+import {STATUS_COLORS} from 'utils/constants';
 
 export default class ActionButton extends React.PureComponent {
     static propTypes = {
         action: PropTypes.object.isRequired,
         handleAction: PropTypes.func.isRequired,
         disabled: PropTypes.bool,
+        theme: PropTypes.object.isRequired,
     }
 
     render() {
-        const {action, handleAction, disabled} = this.props;
+        const {action, handleAction, disabled, theme} = this.props;
+        let customButtonStyle;
+
+        if (action.style) {
+            const hexColor = STATUS_COLORS[action.style] || theme[action.style] || action.style;
```

Comment:
0/5 should we use first the theme color or the STATUS_COLORS (default)?

Reply:
In the merged mobile PR, STATUS_COLORS is used first, so I think we should do the same in the webapp.